tests/quick.rs

```diff
@@ -110,16 +110,16 @@ fn do_ops<K, V, S>(
                     .push(v.clone());
             }
             Remove(ref k) => {
-                evmap.empty(k.clone());
+                evmap.remove_entry(k.clone());
                 write_ref.remove(k);
             }
             RemoveValue(ref k, ref v) => {
-                evmap.remove(k.clone(), v.clone());
+                evmap.remove_value(k.clone(), v.clone());
                 write_ref.get_mut(k).and_then(|values| {
                     values
                         .iter_mut()
                         .position(|value| value == v)
-                        .and_then(|pos| Some(values.remove(pos)))
+                        .and_then(|pos| Some(values.swap_remove(pos)))
```

Comment:
This is unrelated to this PR and is just cleanup from [this](https://github.com/jonhoo/rust-evmap/pull/58#issuecomment-654859029).